content/docs/7.1/deploy/configure/security.md

```diff
@@ -137,11 +141,13 @@ let config = Config::new(/* ... */).with_security(
 );
 ```
 
-Besides, the **connection URL should be changed to `https://`** instead of a plain `ip:port`.
+### [Java Client](https://github.com/tikv/client-java)
+
+Check the [Java client documentation](https://tikv.github.io/client-java/administration/configuration.html#tikvtls_enable)
```

Comment:
How about leave a code snippet here as well ? Just like the Rust client.

Reply:
Make sense.